libs/core/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./model/token";
```

Comment:
Can we change this to only export the specific things we need? https://tkdodo.eu/blog/please-stop-using-barrel-files Stuff like this makes me a little paranoid!

Reply:
Second this, unless you are absolutely certain you need everything from model/token. But even in that case would recommend importing with names as gives a better idea what you're getting.